File: leetcode/2021/January/Practice/java/709-ToLowerCase.java
class Solution {
    public static String toLowerCase(String str) {
		StringBuilder res = new StringBuilder();
		for (char ch : str.toCharArray()) {
			if (ch >= 65 && ch <= 91)
				res.append((char) (ch+32));
			else
				res.append(ch);
		}
		return res.toString();
	}

    public static final void main(String[] args) {
		System.out.println(toLowerCase("LOVELY"));
	}
}

File: leetcode/2021/january/ok/cpp/709-ToLowerCase.cpp
#include <iostream>

using namespace std;

class Solution
{
public:
    string toLowerCase(string str)     {
        for (int i = 0; i < str.length(); i++)
            if (str[i] >= 65 && str[i] <= 91)
                str[i] = str[i] + 32;
        return str;
    }
};

int main() {
    Solution solution;
    string str = "Hello";
    printf("%s", solution.toLowerCase(str).c_str());
    return 0;
}